src/cloud/msg-types.ts

```
@@ -0,0 +1,758 @@
+
+export function MsgIsReqOpen(msg: MsgBase): msg is ReqOpen {
+  return msg.type === "reqOpen";
+}
+
+export interface ResOpen extends MsgBase {
+  readonly type: "resOpen";
+  readonly conn: Connection;
+}
+
+export function buildResOpen(sthis: NextId, req: ReqOpen, resStreamId?: string): ResOpen {
+  if (!(req.conn && req.conn.reqId)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)